### PR TITLE
Add MIRLET7 subfamilies

### PR DESCRIPTION
**Pull request by cthoyt**
- Adds MIRLET7A and its three children
- Adds MIRLET7F and its two children

### CODE REVIEW

**Comment by bgyori** [comment]
Thanks @cthoyt, do you plan to add anything else (the PR is still in Draft stage)?

**Comment by cthoyt** [comment]
What are your overall feelings on including RNA/miRNA families in this resource? It's definitely a place we could add more stuff. I could also make xrefs for this specific entry to miRBase's family resource, for which I have yet to find the underlying data, but am ever vigilant 😂 

If you think we can include this sort of thing, maybe we could do a bit of concerted searching through the INDRA db to find things that start with "miR" or "let" that weren't grounded (or even worse, ones that were grounded with the addition of an "a" or "b" as a suffix that shouldn't be there

**Comment by bgyori** [comment]
@cthoyt what is an example of a miRBase equivalence that we could add? I couldn't find a good entry just by searching.

**Comment by cthoyt** [comment]
The miRBase family for let7 (MIPF0000002) is summarized here: http://www.mirbase.org/cgi-bin/mirna_summary.pl?fam=MIPF0000002 but it's not really obvious how to find this page, and I don't know where there's a listing of all of the families

File: equivalences.csv [hunk 1]
```diff
@@ -1672,3 +1673,4 @@ SIGNOR,SIGNOR-PF6,AMPK_alpha
 SIGNOR,SIGNOR-PF7,p14_3_3
 SIGNOR,SIGNOR-PF8,FGF
 SIGNOR,SIGNOR-PF9,BMP_receptor
+mirbase.family,MIPF0000002,MIRLET7
```
Comment (cthoyt) [inline equivalences.csv:1676]: is this the way you want to stylize it?

Comment (bgyori) [reply equivalences.csv:1676]: It's probably arbitrary but we have had a convention to capitalize these name spaces so maybe we could do that. Having said that I'm tempted to, at some point, standardize equivalences here to use proper curie prefixes, which I am sure you would be pleased about :)

Comment (cthoyt) [reply equivalences.csv]: Addressed with https://github.com/sorgerlab/famplex/pull/134/commits/b43db8904cb8246fb8db3e907ccdb2f6dc72c220 and new export with https://github.com/sorgerlab/famplex/pull/134/commits/0ed9f4a6770bd0104443ed72396051e0f1700041. Sorry for the delay

Comment (bgyori) [reply equivalences.csv]: Thanks!

**Pull request merged**